generator/src/main/java/com/scottlogic/deg/generator/validators/GenerationConfigValidator.java

```diff
@@ -19,6 +23,18 @@ public ValidationResult validateCommandLine(GenerationConfig config) {
             errorMessages.add("RANDOM mode requires max row limit: use -n=<row limit> option");
         }
 
+        if (outputFileAlreadyExists(fileOutputTarget)) {
+            errorMessages.add("Output file already exists, please use a different output filename");
```

Comment:
This message should describe the `--force` switch to allow the file to be overwritten, or is that coming in a later part of the issue?

Reply:
I mean the `--overwrite` command line switch

Reply:
error messages updated to mention `--overwrite` option